pages/_schedule.vue

```diff
@@ -48,10 +48,20 @@ export default {
       isDark: state => state.theme.isDark,
     }),
   },
-  async fetch({ store }) {
-    if (!process.static && !!store.state.splus.schedule) {
-      // not for static build because it depends on the current timestamp
-      await store.dispatch('splus/load');
+  async fetch({ store, params }) {
```

Comment:
Wer ruft eigentlich diese async Funktion auf?

Reply:
Nuxt! https://nuxtjs.org/api/pages-fetch#the-fetch-method